cmd/check_container.go

```diff
@@ -197,6 +197,11 @@ var checkContainerCmd = &cobra.Command{
 			if err != nil {
 				return err
 			}
+
+			log.Info("results successfully submitted to Red Hat")
+			//TODO: displaying this to the user for now so they know submission has be complete
+			// once we get new API contract with actually info to give to the user we can update this
+			fmt.Println("results successfully submitted to Red Hat")
```

Comment:
I'm not sure how I feel about this. We are outputting the test results to stdout as json. If we mix in other output, then that negates the ability to use a pipe to interpret said results. You would then have malformed json. This should either be a log message that always prints in our default loglevel, or it needs to just go to stderr.

Reply:
This was just going to be temporary, but after this morning, we know what to display to the user.

Reply:
So, we definitely do not want to print out to stdout. That should be reserved for the result json. Any output should come in the form of a log.Info. If our default log level is not log.Info, it should be made as such, and the log.Info log level should be reserved for actual messages to the user. It should not be used for error reporting, or debug-level messages.